src/main/java/com/dev/cinema/anotation/FieldsValueMatch.java

```diff
@@ -0,0 +1,31 @@
+package com.dev.cinema.anotation;
+
+import com.dev.cinema.validator.FieldMatchValidator;
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Target({ ElementType.TYPE })
+@Retention(RetentionPolicy.RUNTIME)
+@Constraint(validatedBy = FieldMatchValidator.class)
+public @interface FieldsValueMatch {
+    String message() default "Passwords dont";
```

Comment:
```suggestion
    String message() default "Fields don't match";
```
я так розумію, що ідея була в тому, щоб створити загальний валідатор полів, якщо так, то думаю краще буде змінити меседж. 
Хоча це наврядчи буде частим юз кейсом(порівнювати два поля одного класу), тому тут було б достатньо реалізувати це як PasswordsMatchValidator і параметризувати через UserRegistrationDto/UserRequestDto